src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDescriptorProvider.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.AspNet.Razor.TagHelpers
     /// </summary>
     public class TagHelperDescriptorProvider
     {
-        private const string CatchAllDescriptorTarget = "*";
+        public const string CatchAllDescriptorTarget = "*";
```

Comment:
if this is for tag helper authors, it shouldn't be buried here.  those authors don't know anything about `TagHelperDescriptor` or this provider.  I suggest this live (or be duplicated) in `TargetElementAttribute`.  pretty sure
``` c#
public const string CatchAllDescriptorTarget = TagHelperDescriptorProvider.CatchAllDescriptorTarget;
```
would work in that class.